ee/clickhouse/queries/test/__snapshots__/test_paths.ambr

```diff
@@ -3704,7 +3704,7 @@
                    AND (event = '$pageview')
                    AND NOT path_item IN ['/bar/*/foo']
                    AND toTimeZone(timestamp, 'UTC') >= toDateTime('2012-01-01 00:00:00', 'UTC')
-                   AND toTimeZone(timestamp, 'UTC') <= toDateTime('2023-05-24 23:59:59', 'UTC')
+                   AND toTimeZone(timestamp, 'UTC') <= toDateTime('2023-05-25 23:59:59', 'UTC')
```

Comment:
I think we have somewhere introduced a test without frozen time, since this is showing up in multiple PRs.